pkg/server/sandbox_run_unix.go

```diff
@@ -136,7 +136,7 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 		defer func() {
 			if retErr != nil {
 				// Teardown network if an error is returned.
-				if err := c.teardownPod(id, sandbox.NetNSPath, config); err != nil {
+				if err := c.teardownPod(ctx, id, sandbox.NetNSPath, config); err != nil {
```

Comment:
do we have to keep the `*_unix` files, or can we just delete them all and save ourselves the trouble?

Reply:
I think we should keep them for now while we decide what we should do with our cri fork